.circleci/config.yml

```diff
@@ -522,13 +522,14 @@ jobs:
 
   iOS build and test:
     macos:
-      xcode: "12.3.0"
+      xcode: "12.4.0"
     steps:
       - checkout
       - run:
           name: Install new carthage
           command: |
-            brew upgrade carthage
+            time brew update
```

Comment:
time?

Reply:
Wanted to know the time it takes without splitting this up into circleci tasks. Probably gonna remove this before landing, but right now it's still broken anyway for other reasons now.

Reply:
Gotcha!